chain-impl-mockchain/src/testing/genesis/genesis_tests.rs

```diff
@@ -1,14 +1,16 @@
-use crate::common::ledger as ledger_mock;
+#![cfg(test)]
```

Comment:
That makes it work, but now the unit tests are part of the `testing` module and their modules are also public, even though they won't be compiled for any users of crate.

The `#[quickcheck]` unit tests should be moved to `mod tests` in the library source, next to the types they are testing.